woocommerce-services.php

```diff
@@ -1071,26 +1049,12 @@ public function woocommerce_shipping_methods( $shipping_methods ) {
 			$shipping_service_ids = $this->get_service_schemas_store()->get_all_shipping_method_ids();
 
 			foreach ( $shipping_service_ids as $shipping_service_id ) {
-				$shipping_methods[ $shipping_service_id ] = $this->get_service_object_by_id( 'WC_Connect_Shipping_Method', $shipping_service_id );
+				$shipping_methods[ $shipping_service_id ] = new WC_Connect_Shipping_Method( $shipping_service_id, $shipping_methods );
 			}
 
 			return $shipping_methods;
 		}
 
-		/**
-		 * Registers shipping methods for use in things like the Add Shipping Method dialog
-		 * on the Shipping Zones view
-		 *
-		 */
-		public function woocommerce_load_shipping_methods() {
```

Comment:
Why do we no longer need to load shipping methods?

Reply:
Nope, it's all good in this PR. I am just not familiar with the different filters so wasn't sure what we were removing.